packages/dependency-extraction-webpack-plugin/package.json

```
@@ -29,7 +29,10 @@
 	],
 	"main": "lib/index.js",
 	"exports": {
-		".": "./lib/index.js",
+		".": {
+			"default": "./lib/index.js",
+			"types": "./lib/types.d.ts"
```

`default` should be the last item in the list.


```suggestion
			"types": "./lib/types.d.ts",
			"default": "./lib/index.js"
```